### PR TITLE
Daily notification 

### PR DESCRIPTION
**Pull request by bishownath-d**

### CODE REVIEW

**Comment by bishownath-d** [comment]
Daily notification is set to 60 seconds for testing purpose. Hour and minutes has been commented in the code. Please ignore or if that is not required, I can delete it later. Thank you.

File: src/screens/TabScreen/index.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ const Tab = createBottomTabNavigator();
 
 export default function TabScreen(props) {
 
+	const data = [
```
Comment (susomshrestha) [inline src/screens/TabScreen/index.js:16]: is this used anywhere? Even if used do we need to send it from this screen?

File: src/screens/NotificationScreen/index.js [hunk 2]
```diff
@@ -1,8 +1,48 @@
...
+                    headerShown: false
+                }}
+                name='Notification'
+            >
+                {(props) => {
+                    return (
+                        <Notification
+                            {...props}
+                            words={words}
+                            data={data}
```
Comment (susomshrestha) [inline src/screens/NotificationScreen/index.js:23]: Not used in the notification component but still is being sent. Is is intended?

File: src/screens/NotificationScreen/index.js [hunk 3]
```diff
@@ -1,8 +1,48 @@
...
+
+            <Stack.Screen
+                options={{
+                    headerShown: false
+                }}
+                name="RadioButton"
+            >
+                {(props) => {
+                    return (
+                        <RadioButton
```
Comment (susomshrestha) [inline src/screens/NotificationScreen/index.js:39]: Is this a new component? Cannot find it in main branch and not in this pr.

File: src/screens/TabScreen/index.js (hunk 1 above)
Comment (bishownath-d) [reply src/screens/TabScreen/index.js]: I passed it from tab screen at first but I used it in notification only to change the colour of the notification icon so I added it directly to the notification file. I forgot to remove this one but I already remove it now. Thank you.

File: src/screens/NotificationScreen/index.js (hunk 2 above)
Comment (bishownath-d) [reply src/screens/NotificationScreen/index.js:23]: Removed from here already

**Review by susomshrestha — APPROVED** [review]

**Review by Yadab-Neupane — APPROVED** [review]

**Pull request merged**